test/test_nn.py

```diff
@@ -3196,8 +3196,7 @@ def test_pad_scalar_error(self):
         self.assertRaises(AssertionError, lambda: F.pad(inputs, (1, 1)))
         self.assertRaises(AssertionError, lambda: F.pad(inputs, (1,)))
 
-    @unittest.skipIf((not TEST_NUMPY) or (not TEST_SCIPY) or (scipy.__version__ < '1.0.0'),
```

Comment:
Do we still have `TEST_SCIPY` decorator now?

Reply:
I'm sorry, I don't understand the question. I deleted TEST_SCIPY because scipy is not used in this test